packages/blade/src/components/Chip/Chip.tsx

```diff
@@ -173,7 +186,7 @@ const _Chip: React.ForwardRefRenderFunction<BladeElementRef, ChipProps> = (
                 justifyContent="center"
                 alignItems="center"
                 overflow="hidden"
-                backgroundColor={chipBackgroundColor}
+                backgroundColor={isHovered ? 'transparent' : chipBackgroundColor}
```

Comment:
there's a flicker that happens because we change this on hover. 

https://github.com/razorpay/blade/assets/11384858/a625f3d6-061a-48f1-9e92-213937dc4435

Reply:
Fixed, can you check again.